test/e2e/storage/csi_volumes.go

```diff
@@ -78,13 +78,44 @@ var _ = Describe("PMEM Volumes", func() {
 	// List of testSuites to be executed in below loop
 	var csiTestSuites = []func() testsuites.TestSuite{
 		// TODO: investigate how useful these tests are and enable them.
+
 		// testsuites.InitMultiVolumeTestSuite,
+		// === above line refers to tests that are not found, proposing removal line from here
+
 		testsuites.InitProvisioningTestSuite,
+
 		// testsuites.InitSnapshottableTestSuite,
+		// === above line adds one skipped test (as snapshot-related get skipped), proposing removal line from here
+
 		// testsuites.InitSubPathTestSuite,
+		// === above line changes from "Ran 39 of  83 Specs in 89.985  sec, 39 Passed | 0 Failed | 0 Pending | 44 Skipped"
+		// ===                    to   "Ran 57 of 137 Specs in 925.305 sec, 55 Passed | 2 Failed | 0 Pending | 80 Skipped"
+		// === errors:
+		// === [It] should unmount if pod is force deleted while kubelet is down [Disruptive][Slow]
+		//     No external address for pod pod-subpath-test-pmem-csi-dynamicpv-k559 on node pmem-csi-clear-govm-worker2
+		//
+		// === [It] should unmount if pod is gracefully deleted while kubelet is down [Disruptive][Slow]
+		//     No external address for pod pod-subpath-test-pmem-csi-dynamicpv-q9sj on node pmem-csi-clear-govm-worker3
+		// === proposing checking/fixing errors if possible and enabling, (but adds long time to testing)
+
 		// testsuites.InitVolumeIOTestSuite,
+		// === above line changes from "Ran 39 of 83 Specs in 89.985  sec, 39 Passed | 0 Failed | 0 Pending | 44 Skipped"
+		// ===                    to   "Ran 40 of 86 Specs in 130.594 sec, 39 Passed | 1 Failed | 0 Pending | 46 Skipped"
+		// === error:  [It] should write files of various sizes, verify size, validate content [Slow]
+		// ===         size of file /opt/pmem-csi_io_test_volumeio-1976-104857600 is 1843200, expected 104857600
+		// === proposing checking/fixing errors if possible and enabling, adds 40 sec
+
 		// testsuites.InitVolumeModeTestSuite,
-		// testsuites.InitVolumesTestSuite,
+		// === above line changes from "Ran 39 of 83 Specs in 89.985  sec, 39 Passed | 0 Failed | 0 Pending | 44 Skipped"
+		// ===                    to   "Ran 41 of 87 Specs in 126.549 sec, 40 Passed | 1 Failed | 0 Pending | 46 Skipped"
+		// === error: [It] should fail in binding dynamic provisioned PV to PVC
+		// === proposing checking/fixing errors if possible and enabling, adds 36 sec
+
+		testsuites.InitVolumesTestSuite,
+		// === above line changes from "Ran 39 of  83 Specs in 89.985  sec, 39 Passed | 0 Failed | 0 Pending | 44 Skipped"
+		// ===                    to   "Ran 41 of 113 Specs in 161.006 sec, 41 Passed | 0 Failed | 0 Pending | 72 Skipped"
+		// === No errors, many skipped tests added (does not support preprovisionedPV, inlineVOlume)
+		// === should enable?, adds 2 OK tests (adds 70 seconds)
```

Comment:
Agreed.

Reply:
on a closer look, 2 tests which are not skipped in InitVolumesTestSuite:
- "should be mountable" : starts a pod and checks file on volume has certain content
- "should allow exec of files on the volume" : checks that a script is written and executed in pod
do not add much value in PMEM-CSI case.
I propose we do not add that test suite.